<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReconstructionWizard</class>
 <widget class="QWizard" name="ReconstructionWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconstruction Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <widget class="QWizardPage" name="DeviceAlgorithmPage">
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>209</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Algorithm</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>160</width>
          <height>261</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="sizeConstraint">
          <enum>QLayout::SetNoConstraint</enum>
         </property>
         <item>
          <widget class="QRadioButton" name="rbAlgorithmCGLS">
           <property name="text">
            <string>CGLS</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">btnGrpAlgorithm</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAlgorithmMLEM">
           <property name="text">
            <string>MLEM</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">btnGrpAlgorithm</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAlgorithmSIRT">
           <property name="text">
            <string>SIRT</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">btnGrpAlgorithm</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAlgorithmTikhonov">
           <property name="text">
            <string>CGLS/Tikhonov</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">btnGrpAlgorithm</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbAlgorithmTV">
           <property name="text">
            <string>CGLS/TV</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">btnGrpAlgorithm</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="chkBoxHyperThreads">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>569</y>
      <width>321</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>No Hyper-threads</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>359</width>
      <height>199</height>
     </rect>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="btnDevice">
        <property name="text">
         <string>XTek</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="FileSelectionPage">
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <widget class="QPushButton" name="btnSelectInputFile">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leInputFile">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>50</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="ParametersPage">
   <attribute name="pageId">
    <string notr="true">2</string>
   </attribute>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>501</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pixels per Voxel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBoxPixelsPerVoxel">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Number Of Iterations</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBoxNoOfIterations">
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="chkBoxBeamHardening">
       <property name="text">
        <string>Beam Hardening</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="ProgressWizardPage" name="ProgressPage">
   <attribute name="pageId">
    <string notr="true">3</string>
   </attribute>
   <widget class="QLabel" name="lblProgressTitle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress Title</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pbReconstruction">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>511</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblProgressInfo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Progress Information</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbProgressInfo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>491</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="ResultsPage">
   <attribute name="pageId">
    <string notr="true">4</string>
   </attribute>
   <widget class="QPushButton" name="btnSelectOutput">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Format</string>
    </property>
    <widget class="QRadioButton" name="rbTiffOutputFormat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Unsigned 16-bit tiff</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">btnGrpOutputFormat</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="rbRawOutputFormat">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Floating point data(RAW)</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">btnGrpOutputFormat</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLineEdit" name="leOutputName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="SavingWizardPage" name="SavingProgressPage">
   <widget class="QProgressBar" name="pbSavingProgress">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>501</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblSavingInfo">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>185</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProgressWizardPage</class>
   <extends>QWizardPage</extends>
   <header>ProgressWizardPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SavingWizardPage</class>
   <extends>QWizardPage</extends>
   <header>SavingWizardPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btnGrpAlgorithm"/>
  <buttongroup name="btnGrpOutputFormat"/>
 </buttongroups>
</ui>
